moveit_experimental/moveit_jog_arm/test/test_halt_msg/test_jog_arm_halt_msg.py

```diff
@@ -18,8 +18,8 @@
 
 CARTESIAN_JOG_COMMAND_TOPIC = 'jog_server/delta_jog_cmds'
 
-# jog_arm should publish 'true' here if it halts
-HALT_TOPIC = 'jog_server/halted'
+# jog_arm should publish a string warning here if it halts
+HALT_TOPIC = 'jog_server/status'
```

Comment:
Should this be just `WARNING_TOPIC` or `STATUS_TOPIC` to be more general?

Reply:
yeah, `STATUS_TOPIC` to match the config files makes sense